image/Code/histogram.py

```diff
@@ -0,0 +1,33 @@
+from skimage import data, exposure, img_as_float
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.